src/Database/Behaviors/Encryptable.php

```diff
@@ -0,0 +1,19 @@
+<?php namespace October\Rain\Database;
+
+use \October\Rain\Database\ModelTraitBehavior;
+
+/**
+ * Encryptable trait as behaviour
+ *
+ * @package october\database
+ * @author JoakimBo
+ */
+class Encryptable extends ModelTraitBehavior
+{
+    use \October\Rain\Database\Traits\Encryptable;
+
+    public function __construct($model)
```

Comment:
You don't need the `__construct()` method in any of these behaviors because the ModelTraitBehavior takes care of it for you

Reply:
Okay, I included them since I was in the mindset that we would override their boot methods.

Reply:
Right, do it piece meal though